internal/storage/gcs/errors.go

```diff
@@ -35,3 +45,50 @@ type PreconditionError struct {
 func (pe *PreconditionError) Error() string {
 	return fmt.Sprintf("gcs.PreconditionError: %v", pe.Err)
 }
+
+// WrapUnderCommonGCSError converts an error returned by go-sdk into gcsfuse specific common gcs error.
+// It returns wrapped error and a boolean indicating whether a wrapped or not.
```

Comment:
Nit: I think we should call it handled error instead if we decide to keep it. WDYT?
```suggestion
// It returns wrapped error and a boolean indicating whether error was handled or not.
```

Reply:
Not required, as removed the second wrapped field.